layers.go

```diff
@@ -1180,6 +1192,7 @@ func newROLayerStore(rundir string, layerdir string, driver drivers.Driver) (roL
 		mountsLockfile: nil,
 		rundir:         rundir,
 		jsonPath: [numLayerLocationIndex]string{
+			filepath.Join(layerdir, "layers.json"),
 			filepath.Join(layerdir, "layers.json"),
```

Comment:
The code does not anticipate this. Saving would be completely broken (although that shouldn’t happen for the RO store, sure), and loads call this invalid (“There should be no duplicated ids between json files, but lets check to be sure”).

Reply:
we end up here only when setting up the additional stores, so there is no split of `image store` and `container store` there

Reply:
That doesn’t make the consumers of `jsonPath` any more ready to accept such inputs.

To be explicit, I think the situation should be modeled precisely the way we mean it: perhaps make the `jsonPath` entry `""` (or turn `jsonPath` into a map with missing entries?), and then handle that situation in all consumers (skipping loads, refusing saves, and so on.)

Reply:
used "" in this case and changed the users of `jsonPath` to ignore that entry